src/Tribe/Admin/Notice/Timezones.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Shows an admin notice for Timezones
+ * (When using UTC and on TEC Pages or WordPress > General Settings)
+ */
+class Tribe__Events__Admin__Notice__Timezones {
+
+	public function hook() {
+
+		// Bail if the site isn't using UTC
+		if ( ! $this->is_utc_timezone() ) {
+			return;
+		}
+
+		// Bail if we're not on TEC pages or WordPress General Settings
+		if ( ! $this->should_display() ) {
+			return;
+		}
+
+		tribe_notice(
+			'tribe-events-utc-timezone',
+			array( $this, 'notice' ),
+			array(
+				'type'    => 'warning',
+				'dismiss' => 1,
+				'wrap'    => 'p',
+			)
+		);
+
+	}
+
+	/**
+	 * Checks if the site is using UTC Timezone Options
+	 *
+	 * @since  TBD
+	 *
+	 * @return boolean
+	 */
+	public function is_utc_timezone() {
```

Comment:
Looking into `tribe-common` submodule seems to have a function called [`Tribe__Timezones::wp_timezone_string()`](https://github.com/moderntribe/tribe-common/blob/master/src/Tribe/Timezones.php#L70)

Maybe you can use that one insted to make this one more compact such as: 

```php
return 'UTC' === Tribe__Timezones::wp_timezone_string();
```

Or if you are looking any `UTC` derivate

```php
return strpos( Tribe__Timezones::wp_timezone_string(), 'UTC' ) !== false;
```

Reply:
Cool, thanks for the heads up, Cris! 👍